ext/opcache/tests/preload_user_002.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+preload_user sets the process uid
+--INI--
+opcache.enable=1
+opcache.enable_cli=1
+opcache.optimization_level=-1
+opcache.preload={PWD}/preload_user.inc
+opcache.preload_user={ENV:TEST_NON_ROOT_USER}
```

Comment:
I can't actually see this env variable anywhere, is this actually set?

Reply:
Yes, I will have to define this variable in builds if this is merged